File: CRUApp/app/src/main/java/com/example/cruapp/Alumno.kt
package com.example.cruapp

class Alumno(var nombre:String, var sexo: String, var fecha_nacimiento: String,var latitud:String,
             var longitud:String,var url:String ){
    override fun toString(): String {
        return "${this.nombre},${this.sexo},${this.fecha_nacimiento},${this.latitud}" +
                ",${this.longitud},${this.url}"
    }

}

File: Backend_CRUD/CRUD/api/models/Alumno.js
/**
 * Alumno.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {

   nombre:{
     type:'string',
     minLength:3
   },
   sexo:{
     type:'string',
     maxLength:1
   },
   fecha_nacimiento:{
     type:'string'
   },

   latitud:{
    type:'string'
   },
   longitud:{
    type:'string'
   },
   url:{
    type:'string'
   },

  

  },

};

